### ISSUE TITLE
API Key Environment Variable no Longer Working

### ISSUE THREAD

**Issue by TKassis** [gtalarico/airtable-python-wrapper#97]
I used to be able to set my API key as an environment variable as:

set AIRTABLE_API_KEY=123456

This was under version 0.13. I've updated to version 0.15 and now I get the following error when I run exactly the same script:

TypeError: __init__() missing 1 required positional argument: 'api_key'

Has the environment variable name changed with version 0.15? It's not clear from the docs what it should be.

**Comment by gtalarico** [comment]
You are right, env config was remove in latest release.
https://github.com/gtalarico/airtable-python-wrapper/blob/master/HISTORY.md#0150

I removed it from docs but maybe should have added a not about it. Sorry about that!
https://airtable-python-wrapper.readthedocs.io/en/master/authentication.html#airtable-authentication

I removed it because have it provided 2 ways of doing the same thing, but loading env and passing in code is more explicitly is easier for someone reading code to understand what's happening

Luckily it's an easy fix, just use instead
```python
api_key = os.environ["AIRTABLE_API_KEY"]
Airtable(..., api_key=api_key)
```
